### DISCUSSION TITLE
can't find _includes directory

### DISCUSSION THREAD

**Discussion by ThinkingSpade** [mmistakes/minimal-mistakes#2817] [Q&A]
I want to start by saying I am a complete noob to this (Jekyll,coding,github) but I am trying to explore and have fun. 

I started off by  [Minimal Mistakes remote theme starter](https://github.com/mmistakes/mm-github-pages-starter/generate) and it created a bunch of configuration for me except _includes. I was wondering if anybody knows how I can get that added so I can change my footer. Thanks!

**Comment by iBug** [comment]
You can just create it if it doesn't exist.

![image](https://user-images.githubusercontent.com/7273074/107474388-39502500-6bad-11eb-97bd-68daa3995a53.png)

![image](https://user-images.githubusercontent.com/7273074/107474480-613f8880-6bad-11eb-97aa-93dd7e2525ca.png)

![image](https://user-images.githubusercontent.com/7273074/107474507-6bfa1d80-6bad-11eb-9bb6-d82f91f4ebe3.png)
